### ISSUE TITLE
Wake up/Turn on not working after few hours off

### ISSUE THREAD

**Issue by skydado** [bigmoby/fglair_for_homeassistant#95]
## System Information

version | core-2023.12.4
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.11.6
os_name | Linux
os_version | 6.1.63-haos
arch | x86_64
timezone | Europe/Rome
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 5000
Installed Version | 1.33.0
Stage | running
Available Repositories | 1373
Downloaded Repositories | 22

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 11.2
-- | --
update_channel | stable
supervisor_version | supervisor-2023.12.0
agent_version | 1.6.0
docker_version | 24.0.7
disk_total | 439.4 GB
disk_used | 10.1 GB
healthy | true
supported | true
board | generic-x86-64
supervisor_api | ok
version_api | ok
installed_addons | Advanced SSH & Web Terminal (17.0.1), File editor (5.7.0), Duck DNS (1.15.0), NGINX Home Assistant SSL proxy (3.6.0), Home Assistant Google Drive Backup (0.112.1), GoSungrow (3.0.7), Mosquitto broker (6.4.0)

</details>

<details><summary>Dashboards</summary>

dashboards | 4
-- | --
resources | 16
views | 21
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | December 29, 2023 at 16:47
-- | --
current_recorder_run | January 7, 2024 at 11:00
estimated_db_size | 1008.52 MiB
database_engine | sqlite
database_version | 3.41.2

</details>

<details><summary>Xiaomi Miot Auto</summary>

component_version | 0.7.14
-- | --
can_reach_server | ok
can_reach_spec | ok
logged_accounts | 1
total_devices | 2

</details>

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [previous issues](https://github.com/bigmoby/fglair_for_homeassistant/issues?q=is%3Aissue+label%3A%22Bug%22+)..

### Describe the issue

I have been successfully using this integration with my 2 Fujitsu ARXG24KMLA since September.
I am now trying to move my on/off scheduling from the FGLair app to an HA script.

No problems for the off-scheduling.

But the on-scheduling only works if the switch off is quite recent.
So no problems when I test it (turn off, wait few minutes, turn on).
But there is no way to make it work if several hours have passed since it was turned off, neither with the script nor manually from the UI climate card.
My typical case: the script turn them off at 22 and then (try to) turn them on at 6.

This is the script service call:
`service: climate.turn_on
target:
  entity_id:
    - climate.condiz_zona_giorno
data: {}`

I also tried changing temperature and other settings to "wake up" the devices, but no way.
As I said, I believe it's not a script problem, because they do not wake up even manually changing settings from the UI card too.
When I turn them on again from the FGLair app, they immediatly turn on.

### Reproduction steps

1. turn device off
2. wait few hours
3. turn device on from script or manually
...

### Debug logs

```text
not an expert, but I did not find any useful log in the system log, even with debug enabled
```

**Comment by Z0tteke** [comment]
It looks like i got this issue too. But i can use my AC normally at any given time using the card on the HA dashboard. 
Only thing is it will never turn on at the automation i have set at 6am. But running the automation myself or setting it a few minutes after the current time does work. It only refuses to do so at 6am (and 6:05 and 6:10 am since i tried). 

Did you find a solution?

**Comment by skydado** [comment]
Yes, after many failed attempts.
Don’t try to power on it but just set the HVAC mode that you desire (e.g. HEAT), wait few seconds (e.g. 10s) and then set the target temperature. It is working fine since months.
Let me know if this worked also for you.

**Comment by Z0tteke** [comment]
Great! Gonna try it!
And just to be sure. I had now 2 actions. One was turning on the AC and the other one was setting temp and hvac mode. 
But the solution is to delete those 2 and first start with set hvac mode and a seperate action to set the temp, right?

**Comment by skydado** [comment]
Yes, 2 separated actions in the same automation/script, with a delay between them.

**Comment by Z0tteke** [comment]
Wow! It worked! :)
This made me happy! :D

**Comment by Z0tteke** [comment]
> I wonder if my issue is related. I get Fglair entities, works fine, then after a while they go blank and I get new MQTT entities. They make the AC's work, once I turned the MQTT entities on and off, the Fglair entities populate again for a few hours. Then they go blank again.

That's not something i encountered with my issue. I also don't have an idea to fix this.
Did you try to delete them and add them again?

**Comment by SgtFlippy** [comment]
I deleted the MQTT entities a while back and everything stayed working a while. After a few days I noticed the Fglair cards were empty again and the MQTT entities were back.

**Comment by bigmoby** [comment]
The workaround has been found here: https://github.com/bigmoby/fglair_for_homeassistant/issues/95#issuecomment-2002320074

**Comment by skydado** [comment]
So, I deduce that the climate.turn_off is not working anymore due to some changes at HA OS level, because I did the upgrade more or less at the same time of the FG integration upgrade.

Workaround: instead of using that service to turn off the climate, you can use the "set hvac mode" service like this:

```
service: climate.set_hvac_mode
data:
  hvac_mode: "off"
target:
  entity_id: climate.condiz_zona_giorno
```

**Comment by bigmoby** [comment]
@skydado please try with just released version v0.3.6 I hope it fixs
